content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -152,6 +152,30 @@ host's file owner/group.
 
 [CVE-2021-25741]: https://github.com/kubernetes/kubernetes/issues/104980
 
+## Integration into Pod Security Standards
+
+{{< feature-state state="alpha" for_k8s_version="v1.29" >}}
+
+All available [Pod Security Standards (PSS) profiles](/docs/concepts/security/pod-security-standards)
+will relax in a controlled way for pods which enable user namespaces. This
+behavior can be controlled by the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+`UserNamespacesPodSecurityStandards`, which allows an early opt-in for end
+users. Admins have to ensure that user namespaces are enabled by all nodes
+within the cluster if using the feature gate.
+
+In detail the relaxation means that if user namespaces are enabled, then the
+following fields won't be restricted any more because they always have to refer
+to the user inside the container:
```

Comment:
```suggestion
to the user inside the Pod:
```
user namespaces apply at Pod level, right?

Reply:
This seems very grey to me. Yes, userns are applied for all containers in a pod. But the UIDs are running inside some container. The user is specific to the container. I don't have a strong opinion, really...

I left it as it is for now